src/components/Home.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-
+import dp from '../Images/dp.jpg';
 function Home(props) {
     return (
         <div>
             <h1>Assignment 1</h1>
             <h2>Home page</h2>
+            <img src={dp} alt="dp"/>
```

Comment:
Can you centre the image by setting its margin to auto and add some padding to the top? Also, it would be better if you added a border to it.